google-cloud-clients/google-cloud-contrib/google-cloud-spanner-jdbc/src/test/java/com/google/cloud/spanner/jdbc/it/ITTransactionModeTest.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.spanner.jdbc.it;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import java.util.Arrays;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import com.google.cloud.spanner.ErrorCode;
+import com.google.cloud.spanner.IntegrationTest;
+import com.google.cloud.spanner.Key;
+import com.google.cloud.spanner.Mutation;
+import com.google.cloud.spanner.Statement;
+import com.google.cloud.spanner.jdbc.ITAbstractSpannerTest;
+import com.google.cloud.spanner.jdbc.SpannerExceptionMatcher;
+import com.google.cloud.spanner.jdbc.SqlScriptVerifier;
+
+@Category(IntegrationTest.class)
+@RunWith(JUnit4.class)
+public class ITTransactionModeTest extends ITAbstractSpannerTest {
+  @Rule
+  public ExpectedException exception = ExpectedException.none();
+
+  @Override
+  public void appendConnectionUri(StringBuilder uri) {
+    uri.append("?autocommit=false");
+  }
+
+  @Override
+  public boolean doCreateDefaultTestTable() {
+    return true;
+  }
+
+  @Test
+  public void testSqlScript() throws Exception {
+    SqlScriptVerifier verifier = new SqlScriptVerifier(new ITConnectionProvider());
+    verifier.verifyStatementsInFile("ITTransactionModeTest.sql", SqlScriptVerifier.class, true);
+  }
+
+  @Test
+  public void testDoAllowBufferedWriteInReadWriteTransaction() {
```

Comment:
let's add some additional assertions to these to confirm the writes.

Reply:
Added assertions that the writes were successful.